src/App.test.js

```diff
@@ -1,10 +1,28 @@
-import React from 'react'
-import { fireEvent, render, screen } from '@testing-library/react'
-import App from './App.js'
-import '@testing-library/jest-dom'
+import React from 'react';
+import { fireEvent, render, screen, waitFor } from '@testing-library/react';
+import App from './App.js';
+import '@testing-library/jest-dom';
+import { movieData } from './apiCalls';
+jest.mock('./apiCalls.js');
 
 describe('App', () => {
-    it('should render correctly', () => {
-      
+    it('should render correctly', async () => {
+      movieData.mockResolvedValueOnce({movies: [
```

Comment:
Super glad we figured out we need the `movies: ` specification.  Looking at how the data was structured was the key to the problem we were having.